client/checkout/classic/index.js

```diff
@@ -462,30 +456,13 @@ jQuery( function ( $ ) {
 	}
 
 	// Handle the checkout form when WooCommerce Payments is chosen.
-	const wcpayPaymentMethods = [
-		PAYMENT_METHOD_NAME_BECS,
-		PAYMENT_METHOD_NAME_CARD,
-		PAYMENT_METHOD_NAME_GIROPAY,
-		PAYMENT_METHOD_NAME_SEPA,
-		PAYMENT_METHOD_NAME_SOFORT,
-	];
+	const wcpayPaymentMethods = [ PAYMENT_METHOD_NAME_CARD ];
 	const checkoutEvents = wcpayPaymentMethods
 		.map( ( method ) => `checkout_place_order_${ method }` )
 		.join( ' ' );
 	$( 'form.checkout' ).on( checkoutEvents, function () {
 		if ( ! isUsingSavedPaymentMethod() ) {
-			let paymentMethodDetails = cardPayment;
-			if ( isWCPaySepaChosen() ) {
-				paymentMethodDetails = sepaPayment;
-			} else if ( isWCPayGiropayChosen() ) {
-				paymentMethodDetails = giropayPayment;
-			} else if ( isWCPaySofortChosen() ) {
-				sofortPayment.sofort = {
-					country: $( '#billing_country' ).val(),
-				};
-				paymentMethodDetails = sofortPayment;
```

Comment:
@FangedParakeet Are we deleting these sepa, giropay, and sofort logics because classic checkout no longer needs to handle these payment methods? I am wondering because there is another place with a similar if-then clause. I am also wondering if we can remove the function `isWCPaySepaChosen` itself.

Reply:
I _think_ these functions (`isWCPaySepaChosen`, `isWCPayGiropayChosen`, and `isWCPaySofortChosen`) are completely redundant now, because the test gateways on which they relied have been removed. I think I was just cleaning up the code here, because this branching logic was confusing me a little when I was trying to figure out the chain of flow. 

There are probably related pieces that can also be removed along with this, but I think this file should only be used support card payments now.

Reply:
Perfect! Thanks!